domains/shubhhh.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shubhhhwarrior",
+        "email": "shubhammvaghela999@gmail.com"
+    },
+    "record": {
+        "CNAME": "shubhhh-96.netlify.app/"
```

CNAME cannot contain/

```suggestion
        "CNAME": "shubhhh-96.netlify.app"
```